docs/LV_client/tutorials/Multi viewport.md

```diff
@@ -0,0 +1,24 @@
+# Configure multi viewports
+
+### Multi viewport
+Starting from release **1.12** the user can now configure secondary viewports. Modify the `"listen_port"` on the sensor configuration with any non-zero value to create additional viewports. User can modify the location and pose of the secondary viewports.
```

Comment:
should we make a note somewhere that this is primarily used for HIL, real time systems, direct injection, etc?